pkg/cmd/node/node_ssh.go

```diff
@@ -0,0 +1,134 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/dcos/dcos-cli/pkg/dcos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/dcos/dcos-core-cli/pkg/sshclient"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+func newCmdNodeSSH(ctx api.Context) *cobra.Command {
+	var leader, masterProxy bool
+	var mesosID, proxyIP, privateIP, user, config string
+	var options []string
+
+	cmd := &cobra.Command{
+		Use:   "ssh <command>",
+		Short: "Establish an SSH connection to the master or agent nodes of your DC/OS cluster",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			host, err := detectHost(leader, mesosID, privateIP)
+			if err != nil {
+				return err
+			}
+
+			proxy, err := detectProxy(ctx, masterProxy, proxyIP)
+			if err != nil {
+				return err
+			}
+
+			sshClient, err := sshclient.NewClient(options, config, user, proxy, host)
+			if err != nil {
+				return err
+			}
+
+			return sshClient.Run(ctx, args)
+		},
+	}
+
+	defUser := "core"
+	cluster, err := ctx.Cluster()
+	if err == nil {
+		if dcosConfUser, ok := cluster.Config().Get("core.ssh_user").(string); ok {
+			defUser = dcosConfUser
+		}
+	}
+	cmd.Flags().BoolVar(&leader, "leader", false, "SSH into the leading master")
+	cmd.Flags().BoolVar(&masterProxy, "master-proxy", false, "Proxy the SSH connection through a master node")
+	cmd.Flags().StringVar(&mesosID, "mesos-id", "", "The agent ID of a node")
+	cmd.Flags().StringVar(&proxyIP, "proxy-ip", "", "Proxy the SSH connection through a different IP address")
+	cmd.Flags().StringVar(&privateIP, "private-ip", "", "Agent node with the provided private IP")
+	cmd.Flags().StringVar(&user, "user", defUser, "The SSH user")
+	cmd.Flags().StringVar(&config, "config-file", "", "Path to SSH configuration file")
+	cmd.Flags().StringSliceVar(&options, "option", nil, "The SSH options. For more information, enter `man ssh_config` in your terminal")
```

Comment:
`man ssh_config` is UNIX specific, I think we should either remove it or append this extra hint based on `runtime.GOOS`